static/js/src/advantage/subscribe/react/components/PaymentMethodForm.jsx

```diff
@@ -100,8 +99,8 @@ function PaymentMethodForm({ setCardValid }) {
           });
         }
       },
-      onSuccess: () => {
-        queryClient.invalidateQueries("preview");
+      onSettled: () => {
+        setSubmitting(false);
```

Comment:
Doesn't `setSubmitting(false)` need to be called in `onError`, too?

Reply:
OnSettled is called both on Error and on Success!

Reply:
I see. All good then 👍